library/general/requiredprobes/samples/must-have-probes/constraint.yaml

```diff
@@ -3,10 +3,12 @@ kind: K8sRequiredProbes
 metadata:
   name: must-have-probes
 spec:
+  enforcementAction: warn
```

Comment:
Hi @ctrought  I believe the CI is depending on the default `enforcementAction` mode which is deny.

```suggestion
```

Reply:
Oops my bad - fixed, thanks!